tests/test_mapmessage.py

```diff
@@ -0,0 +1,18 @@
+from tests.output_betterproto.mapmessage import (
+    Nested,
+    Test,
+)
+
+
+def test_mapmessage():
```

Comment:
```suggestion
def test_mapmessage_to_dict_preserves_message():
```

Nitpick, but something like that might be good.